data/localizations/attributes/de.yml

```diff
@@ -1,3 +1,4 @@
+# This file is auto-generated through a github action. Do not edit directly.
```

Comment:
Does this mean public contributors will no longer be able to submit translations suggestions via PR? Would they just do so via an issue then if they have translation suggestions?

Reply:
Yes that's correct. We'll want to confirm any localization with the linguists so we'd prefer to go through an issue.